service/controller.go

```diff
@@ -873,15 +891,22 @@ func (s *service) ControllerPublishVolume(
 		fsID := getFilesystemIDFromCsiVolumeID(csiVolID)
 		fs, err := s.getFilesystemByID(fsID, systemID)
 		if err != nil {
-			if strings.EqualFold(err.Error(), sioGatewayVolumeNotFound) || strings.Contains(err.Error(), "must be a hexadecimal number") {
+			if strings.EqualFold(err.Error(), sioGatewayFilesystemNotFound) || strings.Contains(err.Error(), "must be a hexadecimal number") {
```

Comment:
update the error scenario wrt to volume, to keep the consistency throughout.

Reply:
done